app/konnect/org-management/teams-and-roles/index.md

```diff
@@ -19,52 +19,50 @@ access.
 You can find a list of all teams in your organization through
 {% konnect_icon organizations %} **Organization** > **Teams** in {{site.konnect_short_name}}.
 
-You must be part of the **Organization Admin** team to manage users, teams, and
+You must be part of the `Organization Admin` team to manage users, teams, and
 roles.
 
 * **Team:** A group of users with access to the same roles. Teams are useful
 for assigning access by functionality, as they can provide granular access to
 any group of {{site.konnect_short_name}} resources based on roles. As well, access to specific
-runtime groups and Services can be shared with teams.
+runtime groups and services can be shared with teams.
 
 * **Role:** Predefined access to a particular resource instance, or all
-instances of a resource type (for example, a particular Service or all Services).
+instances of a resource type (for example, a particular service or all services).
 
-When you create a {{site.konnect_short_name}} account, you are automatically added to the Organization
-Admin team, which is one of the [predefined teams](/konnect/org-management/teams-and-roles/teams-reference)
+When you create a {{site.konnect_short_name}} account, you are automatically added to the `Organization
+Admin` team, which is one of the [predefined teams](/konnect/org-management/teams-and-roles/teams-reference)
 in {{site.konnect_short_name}}. Predefined teams have sets of roles that can't be modified or
 deleted. You can add users to these teams, or create your own custom teams
 with any of the [supported roles](/konnect/org-management/teams-and-roles/roles-reference).
 
-All users can view all teams and members of each team, but they can't view the
-team roles.
-
 ### Access precedence
 
 Users can be part of any number of teams, and the roles gained from the teams
-are additive. For example, if you add a user to both the Service Developer and
-Portal Viewer teams, the user can create and manage Services
+are additive. For example, if you add a user to both the `Service Developer` and
+`Portal Viewer` teams, the user can create and manage services
 through the Service Hub _and_ register applications through the Dev Portal.
 
 If two roles provide access to the same entity, the role with more access
-takes effect. For example, if you have the Service Admin and Service Deployer
-roles on the same Service, the Service Admin role takes precedence.
+takes effect. For example, if you have the `Service Admin` and `Service Deployer`
```

Comment:
```suggestion
takes effect. For example, if you have the Service Admin and Service Deployer
```
I'm removing these because I think it is confusing to use code tags to mark something that doesn't have to do with code.